internal/measurexlite/resolver.go

```diff
@@ -0,0 +1,106 @@
+package measurexlite
+
+//
+// Trusted Recursive Resolver
+//
```

Comment:
Either call the file `trr2.go` and keep this comment or call this file `resolver.go` and say it contains resolvers